scripts/rollup-manifest-filter.js

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
please add some comment to the top of this file explaining what it does; I have no idea